content/learn/migration-guides/0.14-to-0.15.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "0.14 to 0.15"
+insert_anchor_links = "right"
+[extra]
+weight = 10
+long_title = "Migration Guide: 0.14 to 0.15"
+public_draft = 1741
++++
```

Comment:
```suggestion
public_draft = 1741
status = 'hidden'
+++
```

I can't remember if this still does anything really, but I know we at least added this for the sitemap in the past to doubly ensure the page isn't made available to search engines.